ehb_service/apps/core/migrations/0001_initial.py

```diff
@@ -1,268 +1,282 @@
 # -*- coding: utf-8 -*-
```

Comment:
for file ebb_service/apps/core/migrations/0001_initial.py can you change 'initial' to indicate this is a product of migrating to Django 1.7 and removing south? I just want to make sure when we look at this again - that we know that this is not the original migration, but the migration for removing south. Can you replace 'initial' with something short and sweet?

Reply:
renamed to: django17Generated_initial b9aefce